references/settings.md

```diff
@@ -44,11 +44,11 @@ List the settings.
 }
 ```
 
-## Add or update settings
+## Add or replace index settings
```

Comment:
```suggestion
## Add or replace the settings related to an index
```

Reply:
Isn't that too long for a title (visible in the sidebar)?
Plus, for all the routes related to an index (documents, stop-word, synonyms, updates) we hadn't put that precision in the titles.
To be consistent, I can remove it from the title if you prefer? :)

Reply:
It's ok to keep the title you wrote, I forget that it was visible in the sidebar.